### ISSUE TITLE
Why my AVA demo result is so poor with pre-trained models from model zoo?

### ISSUE THREAD

**Issue by syenpark** [facebookresearch/SlowFast#653]
I try reproducing the AVA 2.1 demo in [VISUALIZATION_TOOLS.md](https://github.com/facebookresearch/SlowFast/blob/main/VISUALIZATION_TOOLS.md) with pre-trained models [ava/pretrain/SLOWFAST_32x2_R101_50_50_v2.1.pkl](https://dl.fbaipublicfiles.com/pyslowfast/model_zoo/ava/pretrain/SLOWFAST_32x2_R101_50_50_v2.1.pkl) and [ava/SLOWFAST_32x2_R101_50_50_v2.1.pkl](https://dl.fbaipublicfiles.com/pyslowfast/model_zoo/ava/SLOWFAST_32x2_R101_50_50_v2.1.pkl) downloaded from [MODEL_ZOO.md](https://github.com/facebookresearch/SlowFast/blob/main/MODEL_ZOO.md#ava) with following

```
DEMO:
  ENABLE: True
  OUTPUT_FILE: yourPath/output.mp4
  LABEL_FILE_PATH:  yourPath/ava_classnames.json
  INPUT_VIDEO: yourPath/videos/HVAmkvLrthQ.mp4  # Path to a video file or image folder
  PREDS_BOXES: yourPath/ava_detection_train_boxes_and_labels_include_negative_v.2.1.csv
  GT_BOXES: yourPath/ava_train_v2.1.csv # Path to ground-truth boxes and labels in AVA format (optional).
```
in [demo/AVA/SLOWFAST_32x2_R101_50_50.yaml](https://github.com/facebookresearch/SlowFast/blob/main/demo/AVA/SLOWFAST_32x2_R101_50_50.yaml).

However, unlike the [ava_demo2.gif](https://github.com/facebookresearch/SlowFast/raw/main/demo/visualization/demo_gifs/ava_demo2.gif) in this repo,
![Screenshot 2023-05-30 at 14 01 36](https://github.com/facebookresearch/SlowFast/assets/14952045/9b2eca7a-3db5-4b58-9573-b2f7924537d3)

my demo result has mostly no labels as follows.
<img width="752" alt="Screenshot 2023-05-30 at 14 06 45" src="https://github.com/facebookresearch/SlowFast/assets/14952045/65bd15ab-46ce-4821-b86c-875d9a08ff34">

but it sometime has the labels.
<img width="752" alt="Screenshot 2023-05-30 at 14 07 00" src="https://github.com/facebookresearch/SlowFast/assets/14952045/a8b3c5b1-8b49-4f36-992e-8261f99d10a9">

Does anyone guess the reason why my trial is not as good as the demo in this repository? Pre-trained models are not the one showing the repository's demo?

Thank you in advance.

**Comment by syenpark** [comment]
I resolved it by working on two things:
1. Pre-trained model should be from [ava/SLOWFAST_32x2_R101_50_50_v2.1.pkl](https://dl.fbaipublicfiles.com/pyslowfast/model_zoo/ava/SLOWFAST_32x2_R101_50_50_v2.1.pkl), not ava/pretrain/SLOWFAST_32x2_R101_50_50_v2.1.pkl
2. CHECKPOINT_TYPE: pytorch in [demo/AVA/SLOWFAST_32x2_R101_50_50.yaml](https://github.com/facebookresearch/SlowFast/blob/main/demo/AVA/SLOWFAST_32x2_R101_50_50.yaml) should be updated to caffe2
